wasmtime-jni/src/io_github_kawamuray_wasmtime_Store/imp.rs

```diff
@@ -61,4 +61,10 @@ impl<'a> JniStore<'a> for JniStoreImpl {
         store.gc();
         Ok(())
     }
+
+    fn interrupt_handle_ptr(env: &JNIEnv, this: JObject) -> Result<jlong, Self::Error> {
+        let store = interop::get_inner::<Store<StoreData>>(&env, this)?;
+        let interrupt_handle = store.interrupt_handle()?;
```

Comment:
Currently this value never dropped anywhere, which will cause memory leak.
We have to implement `Disposable` to `InterruptHandle` class and implement `dispose()` methods like other classes.